### PR TITLE
Update iot-workshop-deployment-template.json

### PR DESCRIPTION
**Pull request by manashgoswami**
Fixed docker networking issue in the HostConfig field.

### CODE REVIEW

**Comment by manashgoswami** [comment]
@tedway please review and merge this change. Thanks.

File: IoT Edge anomaly detection tutorial/iot-workshop-deployment-template.json [hunk 1]
```diff
@@ -64,7 +64,7 @@
             "restartPolicy": "always",
             "settings": {
               "image": "__REGISTRY_IMAGE_LOCATION",
-			        "createOptions": "{\"HostConfig\":{\"Binds\":[\"/data/misc/camera:/app/vam_model_folder\"],\"NetworkMode\":\"host\"},\"NetworkingConfig\":{\"EndpointsConfig\":{\"host\":{}}},\"WorkingDir\":\"/var/azureml-app\"}"
+              "createOptions": "{\"HostConfig\":{\"Binds\":[]},\"WorkingDir\":\"/var/azureml-app\"}"
```
Comment (emgerner-msft) [inline IoT Edge anomaly detection tutorial/iot-workshop-deployment-template.json]: If you're getting rid of the binds anyways (which I just checked does work), I think you can get rid of the whole HostConfig leaving you just with the WorkingDir setting.

Comment (emgerner-msft) [reply IoT Edge anomaly detection tutorial/iot-workshop-deployment-template.json]: Actually, I just deleted all the createOptions and it seems to be fine...

Comment (manashgoswami) [reply IoT Edge anomaly detection tutorial/iot-workshop-deployment-template.json]: Updated. Thanks.

**Comment by emgerner-msft** [comment]
Looking at the rest of the repo this may have just been copy/pasted from the below. I just tried and I don't think we maybe even need createOptions at all.

https://github.com/Azure/ai-toolkit-iot-edge/blob/81b2e47a945dc89799b62dcbcec8e052673bc3f6/amliotedge/deploymodel#L79

**Comment by emgerner-msft** [comment]
Friendly ping. :) Could someone take a look at this?

**Comment by gopitk** [comment]
Merging #17.

**Pull request merged**